client/signup/jetpack-connect/plans.jsx

```diff
@@ -58,12 +58,11 @@ const Plans = React.createClass( {
 	},
 
 	componentWillReceiveProps( props ) {
-		const selectedSite = props.sites.getSelectedSite();
```

Comment:
oops, no... to be honest, this has been a case of a lousy search/replace 'selectedSite' by 'this.props.selectedSite'.

Anyway, now you mention it, @aduth , there's no reason to do all this in componentWillReceiveProps ... I could very well move it to ComponentDidUpdate and probably will be less confusing (as in nobody asking "why do you need to check all this before the component props are updated?")